src/main/webapp/query-filters/filter/filter-serialization.tsx

```diff
@@ -0,0 +1,96 @@
+import { getDefaultValue } from './filter-utils'
+import { getIn } from 'immutable'
+
+export const deserialize = (filter: any, metacardTypes: any) => {
```

Comment:
This file is the same as the query-advanced serializaiton file was minus the ability to handle filter groups (I felt it was simpler to just deal with it on a per filter basis